openfl/component/aggregator/aggregator.py

```diff
@@ -103,6 +105,7 @@ def __init__(self,
         self.collaborator_tasks_results = {}
         # {TaskResultKey: data_size}
         self.collaborator_task_weight = {}
+        self.tb_writer = SummaryWriter(self.log_dir, flush_secs=10) if tensor_board else None
```

Comment:
Could users pass `SummaryWriter` instance instead of the flag deciding the TensorBoard usage? It would allow users to define the logging directory, flush frequency, and other keyword arguments of `SummaryWriter`. Moreover, in this case, we do not need `tensorboardX` as a dependency.

Reply:
resolve 747b5c53fee5da2143a617db0311e523c7c937d9